test/cluster/conftest.py

```diff
@@ -265,9 +269,31 @@ async def manager(request: pytest.FixtureRequest,
     await manager_client.stop()  # Stop client session and close driver after each test
     if cluster_status["server_broken"]:
         pytest.fail(
-            f"test case {test_case_name} leave unfinished tasks on Scylla server. Server marked as broken,"
+            f"test case {test_case_name} left unfinished tasks on Scylla server. Server marked as broken,"
             f" server_broken_reason: {cluster_status["message"]}"
         )
+    if found_errors:
+        lines = []
+        with open(failed_test_dir_path / "found_errors.txt", "w") as f:
```

Comment:
nit. I would better separate these two things of creating the lines and writing it to the file. It will increase readability with showing that we are doing two things, actually. And a small bonus that it will decrease the nesting.

Reply:
Agreed, you do not write until the last thing in this block, you can postpone opening it only for when you actually need it and keep it opened for less time